cmd/new.go

```diff
@@ -43,6 +45,8 @@ func init() {
 		false,
 		"Print new fragment instead of writing to disk",
 	)
+	newCmd.Flags().StringVarP(&_kind, "kind", "k", "", "Set the change kind")
+	newCmd.Flags().StringVarP(&_body, "body", "b", "", "Set the change body")
```

Comment:
```suggestion
	newCmd.Flags().StringVarP(&_kind, "kind", "k", "", "Set the change kind without a prompt")
	newCmd.Flags().StringVarP(&_body, "body", "b", "", "Set the change body without a prompt")
```
Just a little extra clarity

Reply:
Done.